### ISSUE TITLE
Add Exit Button To All Modals

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1691]
Not all modals have an exit button. Whilst they can be exited with the escape key, for the sake of consistency all modals should have an exit button.
